### DISCUSSION TITLE
New UI update ruined my Unturned graphics.

### DISCUSSION THREAD

**Discussion by MadKevv** [SmartlyDressedGames/Unturned-3.x-Community#2244] [Q&A] [answered]
Whenever I play unturned I get blue and white boxes where the hud used to be. These boxes phase in and out of existence in split seconds. But can still appear one after another to make it look like it's glitching. I also get it when I open my map up. Practically they show up on anything I can click on. It Honestly looks like someone poured water on my PC(Note: It only happens when I play Unturned). Please Help as it really annoys me to keep seeing this. I've done everything to fix it uninstalling, verifying game files, running unturned BE, and clearing my game cache.

**Comment by SDGNelson** [comment, marked as answer]
Here's how to revert to the old UI system:
1. Right-click Unturned in your Steam library
2. Click "Properties..."
3. Click "Select Launch Options..."
4. Add "-Glazier=IMGUI" without quotes

For more info: https://github.com/SmartlyDressedGames/U3-Docs/blob/master/Glazier.md
